libexec/estate-change/metadata.yaml

```diff
@@ -48,4 +48,4 @@ help:
 
     WARNING: If the node is currently powered on, it will be powered off and then powered back on.
 
-confirmation: Change machine type for %{nodes}. If powered on it will be rebooted. Confirm to proceed [y/n]?
+confirmation: Change machine type for %{nodes}. It will be briefly unavailable if power on. Confirm to proceed [y/n]?
```

Comment:
What does "briefly unavailable" mean for azure?  For AWS "rebooted" means you'll lose work.  We need to be clear that that is a possibility for AWS.

Also, `*powered on`.

Reply:
Honestly I do not know what happens during a hot resize. Will need to do some digging

Reply:
Doing some investigating, looks like `azure` powers the machine off and on again automatically. So it essentially the same process as `aws`

Reply:
Can you change this back to the original "If powered on it will be rebooted."  That is much clearer about what will happen and the possibility that work can be lost.